metadata-ingestion/src/datahub/ingestion/source/sql/vertica.py

```diff
@@ -133,6 +133,7 @@ def create(cls, config_dict: Dict, ctx: PipelineContext) -> "VerticaSource":
         return cls(config, ctx)
 
     def get_workunits_internal(self) -> Iterable[Union[MetadataWorkUnit, SqlWorkUnit]]:
+        yield from super().get_workunits_internal()
         sql_config = self.config
         if logger.isEnabledFor(logging.DEBUG):
             # If debug logging is enabled, we also want to echo each SQL query issued.
```

Comment:
this if statement and the one below it can be removed, since they're already in `super().get_workunits_internal()`

Reply:
Done.Thanks harshal for your review. Did the changes according to your comments